honeybee_energy/ventcool/crack_data.py

```diff
@@ -0,0 +1,89 @@
+# coding':utf-8
+"""Crack data dictionary.
+
+This data is obtained from the DesignBuilder Crack Template. Note that this
+crack data is normalized by area and perimeter for face cracks and subface edges,
+respectively. In EP face cracks are not normalized by area, but subface edges
+are normalized by perimeter.
+"""
+
+crack_data_dict = {
+
+    'external_tight_cracks': {
+        # kg/s/m2 @ 1 Pa
```

Comment:
Do you need to keep repeating this comment?  Can't you just make a note if it in the docstring at the top of the page?